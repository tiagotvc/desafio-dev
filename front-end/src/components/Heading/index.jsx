import React from 'react';
import * as Styled from './styles';

export const Heading = ({children}) => {
    return <Styled.Title >{children}</Styled.Title>;
  };

export default Heading