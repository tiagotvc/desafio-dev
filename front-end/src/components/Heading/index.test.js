import React from 'react';
import Heading from './index';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

describe('Heading ', () => {
  it('renders correctly', () => {
    const tree = renderer
    .create(<Heading>Render Test</Heading>)
    .toJSON();
  expect(tree).toMatchSnapshot();
    });
});