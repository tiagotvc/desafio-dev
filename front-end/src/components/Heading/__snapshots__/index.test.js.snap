// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading  renders correctly 1`] = `
.c0 {
  color: black;
}

<h1
  className="c0"
>
  Render Test
</h1>
`;
